clang/lib/CodeGen/ConstantInitBuilder.cpp

```diff
@@ -296,3 +296,21 @@ ConstantAggregateBuilderBase::finishStruct(llvm::StructType *ty) {
   buffer.erase(buffer.begin() + Begin, buffer.end());
   return constant;
 }
+
```

Comment:
@asl Updating to resolve conflict required bringing this function in that was previously in Akira's work

Reply:
Yeah, LGTM!